images/scorecard-test/bin/entrypoint

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh -e
+
+# This is documented here:
+# https://docs.openshift.com/container-platform/3.11/creating_images/guidelines.html#openshift-specific-guidelines
+
+if ! whoami &>/dev/null; then
+  if [ -w /etc/passwd ]; then
+    echo "${USER_NAME:-test}:x:$(id -u):$(id -g):${USER_NAME:-test} user:${HOME}:/sbin/nologin" >> /etc/passwd
+  fi
+fi
```

Comment:
Sorry, I should have been more clear in my last comment about chmod'ing /etc/passwd. I think this was also removed in https://github.com/operator-framework/operator-sdk/pull/2469. That PR should show you the full set of changes that were made to fix the user setup and entrypoint scripts.

Reply:
removed in favor of https://github.com/operator-framework/operator-sdk/pull/2837 which will include those scripts and the Dockerfile.